specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +853,110 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDnsZoneGroup": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
```

Comment:
should it be read-only?

Reply:
I think so, let me double check with Ajinkya

Reply:
They aren't, user provides name

Reply:
Yes. User provides the name by privateDnsZoneGroupName parameter. Does user need to set this name in both path and body?